src/main/java/com/ldtteam/blockout/controls/AbstractTextElement.java

```diff
@@ -268,6 +294,8 @@ else if (textAlignment.isHorizontalCentered())
             lineShift += mc.fontRenderer.FONT_HEIGHT + textLinespace;
         }
 
+        // TODO: forge disable filtering
```

Comment:
todo for the future?

Reply:
todo for this <https://github.com/MinecraftForge/MinecraftForge/pull/7645>

Reply:
kk